Cellpose_Plugin_CellProfiler_Instructions/README.md

```diff
@@ -0,0 +1,97 @@
+# Cellpose Plugin for CellProfiler installation
+
+## For Linux:
+
+### Step 1 
+
+Create a `.yml` file using the template below from the [CellProfiler conda environment wiki page](https://github.com/CellProfiler/CellProfiler/wiki/Conda-Installation):
+
+```
+name: cp4
+
+channels:
+- conda-forge
+- anaconda
+- bioconda
+- defaults
+
+dependencies:
+- python=3.8
+- pip
+- numpy
+- matplotlib
+- pandas
+- mysqlclient
+- openjdk
+- scikit-learn
+- mahotas
+- gtk2
+- Jinja2=3.0.1
+- inflect=5.3.0
+- wxpython=4.1.0
+- mysqlclient=1.4.4
+- sentry-sdk=0.18.0
+- pip:
+- cellprofiler==4.2.4
+```
+
+## Step 2 
+
+You will use this file to create a conda environment (called `cp4`). 
+Use the code below in the terminal to run the file and install CellProfiler:
+
+```console
+conda env create -f cellprofiler_env.yml
+```
+
+## Step 3a
+
+Once the environment is created, activate it by running:
+
+```console
+conda activate cp4
+```
+
+Step 3b: Install Cellpose into your new conda environment. 
+You need Cellpose downloaded to be able to use the module once it has been added to CellProfiler.
+
+```console
+pip install cellpose
+```
+
+## Step 4 
+
+If you have not already, you should clone the CellProfiler repo from Github (I recommend putting it into a folder on your Desktop called `GitHub` for better organization).
```

Comment:
Why include `CellProfiler` in the environment if you're going to clone it from source?

Reply:
Because it is easier to already have it installed from the .yml file in my opinion instead of having to go through cloning, checking out a specific commit for 4.2.4, and then using `pip install -e .`. I feel like there is more room for error this way.

By having it already installed, then all the individual has to do is have the cloned repo to put the plugin into, then change the path in CellProfiler to that repo.